uber/models.py

```diff
@@ -614,35 +633,6 @@ def match_to_group(self, attendee, group):
                     self.add(attendee)
                     self.commit()
 
-        def everything(self, location=None):
```

Comment:
It's dead!

This method really had no reason to exist once we changed to SQLAlchemy.  It originally only existed to perform some performance tuning when we were using Django (or at least that's the original genesis of this kind of function).  Ah well, I guess it's all academic now.